MultiSnap_timelapse_ef_230209_3.bsh

```diff
@@ -125,7 +126,8 @@ for(int qy=0;qy<panelnumy;qy++){
 
 			cImg = img.copyAtCoords(coord.build());
 	 		StoreWide.putImage(cImg);
-	 		DisplasyWide = mm.displays.show(StoreWide); //show saved wide field images gy230211
+	 		//DisplasyWide = mm.displays().show(StoreWide); //show saved wide field images gy230211
+	 		//DisplaysWide = mm.displays().show(StoreWide);
```

Comment:
Thanks for your checking. I think these could be deleted.

Reply:
Yes, these could be deleted.